src/Components/ZendeskWrapper.tsx

```diff
@@ -21,6 +21,12 @@ export const ZendeskWrapper: FC<ZendeskWrapperProps> = ({
     src: `https://static.zdassets.com/ekr/snippet.js?key=${key}`,
     removeOnUnmount: true,
     onReady: () => {
+      window.zESettings = {
+        webWidget: {
+          // keeping the widget below the cookie banner
+          zIndex: 99,
```

Comment:
I'd set this to `1`

Reply:
I _do_ wish we were able to trigger this with our own UI. Did you find out if that was possible?

Reply:
Yep! Afaik We can change the basic look of the button (image, text, color) 
I think that should be a separate ticket, though!

Reply:
Oh, I'm looking for more than that. Like I want us to have our own component trigger it; rather than pass it styles. But sure.